CONTRIBUTING.md

```diff
@@ -8,152 +8,104 @@
 <!-- TODO: code coverage -->
 </p>
 
-# Contributing to Versatile Data Kit
+# How to Contribute
+Contributing to an open source project is a great way to build skills, make connections, and gain experience.
+We welcome many different types of contributions that include:
+- Adding new features and proposals
```

Comment:
nit: A lot of people really like writing design specifications

Reply:
Also, creating designs and visuals is welcome.

Reply:
Added contribution types.